src/clients/inst_root.rb

```diff
@@ -168,18 +162,8 @@ def main
         )
       )
 
-      if @check_CA_constraints
-        @helptext = Ops.add(
-          @helptext,
-          Builtins.sformat(
-            # additional help text about password
-            _(
-              "<p>If you intend to use this password for creating certificates,\nit has to be at least %1 characters long.</p>"
-            ),
-            @pw_min_CA
-          )
-        )
-      end
+      validator = Users::CAPasswordValidator.new
+      @helptext << validator.help_text if validator.enabled?
```

Comment:
what about tell, not ask and simply just use `@helptext << validator.help_text` and text_help method returns "" if not enabled?